build-scripts/config_common.cmake

```diff
@@ -338,6 +338,9 @@ endif ()
 if (DEFINED WAMR_BH_VPRINTF)
   add_definitions (-DBH_VPRINTF=${WAMR_BH_VPRINTF})
 endif ()
+if (DEFINED WAMR_BH_LOG)
+  add_definitions (-DBH_LOG=1)
```

Comment:
Suggest to be similar with WAMR_BH_VPRINTF:
```cmake
if (DEFINED WAMR_BH_LOG)
  add_defintions (-DBH_LOG=${WAMR_BH_LOG})
endif ()
```
So we only introduce macro BH_LOG to the code, and can use it as a flag to control the code, and developer can define the function with the name he wants.
In bh_log.h:
```
#ifndef BH_LOG
void
bh_log(LogLevel log_level, const char *file, int line, const char *fmt, ...);
#else
void
BH_LOG(uint32 log_level, const char *file, int line, const char *fmt, ...);
#define bh_log BH_LOG
#endif
```

We can also use WASM_LOG, e.g. `add_defintions (-DWASM_LOG=${WAMR_BH_LOG})`, but it seems that BH_LOG is better.

Reply:
Thanks, good point, updated